### PR TITLE
chore(confd): added confd env support

### PR DESCRIPTION
**Pull request by CL-SHLOMIKONCHA**

### CODE REVIEW

File: confd/dev/conf.d/development.toml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+[template]
+src = "production.tmpl"
+dest = "config/default.json"
+keys = [
+    "/configuration/server/port",
```
Comment (asafMasa) [inline confd/dev/conf.d/development.toml:5]: configuration

File: confd/dev/templates/production.tmpl [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{
+    "server": {
```
Comment (asafMasa) [inline confd/dev/templates/production.tmpl:2]: add confd.exe to git ignore

Comment (asafMasa) [reply confd/dev/templates/production.tmpl:2]: confd/dev git ignore

File: config/default.json [hunk 3]
```diff
@@ -1,19 +1,18 @@
...
-    "dynamicActivityFields": ["action", "method", "additional"],
-    "dynamicActivityNameValues": ["apiInvoker"],
-    "ingestFields": ["imageMetaData", "action"]
-  },
-  "logger": {
-    "level": "info",
-    "log2file": true
-  }
+    "server": {
+        "port": 3000
```
Comment (asafMasa) [inline config/default.json:3]: delete file and add to git ignore since its generated

File: confd/dev/conf.d/development.toml (hunk 1 above)
Comment (CL-SHLOMIKONCHA) [reply confd/dev/conf.d/development.toml:5]: canceled

File: confd/dev/templates/production.tmpl (hunk 2 above)
Comment (CL-SHLOMIKONCHA) [reply confd/dev/templates/production.tmpl]: fixed in 198e01c

File: config/default.json (hunk 3 above)
Comment (CL-SHLOMIKONCHA) [reply config/default.json:3]: fixed in 9943996

**Pull request merged**